### PR TITLE
(doc) Readme update for CICS support

### PR DESCRIPTION
**Pull request by zeibura**

### CODE REVIEW

**Review by zimlu02 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Autocomplete speeds up the coding process by intuitively suggesting the most lik
 - Code Snippets
 - Copybook variable and paragraph names
 - Names of copybooks that are used in the program
+- CICS keywords inside `EXEC CICS` statements
```
Comment (temanbrcom) [inline README.md:47]: I think that specifying "inside EXEC CICS" is misleading - we don't check where the complete suggestions were requested from, so every time we return the CICS keywords together with COBOL ones.

File: README.md [hunk 2]
```diff
@@ -52,6 +53,8 @@ The autocomplete feature is only available in the main COBOL file, not in copybo
 ### Syntax and Semantic Check for Code
 This feature checks for mistakes and errors in COBOL code. The syntax check feature reviews the whole content of the code and suggests fixes, through syntax and semantic analysis which returns diagnostics on the entire context of the code, not just keywords.
 
+This feature is also enabled for CICS keywords inside `EXEC CICS` statements.
```
Comment (temanbrcom) [inline README.md:56]: Here would be better "CICS keywords and variables"

File: README.md (hunk 1 above)
Comment (zeibura) [reply README.md]: changed in latest commit

File: README.md (hunk 2 above)
Comment (zeibura) [reply README.md]: changed in latest commit

**Review by temanbrcom — APPROVED** [review]
🦖

**Pull request merged**